docs/pages/keyconcepts/templating/templating.rst

```diff
@@ -0,0 +1,39 @@
+.. _matching:
+
+
+Templating
+----------
+
+Hoverfly can build responses dynamically through templating. This is particularly useful when combined with loose matching, as it allows a single
+matcher to represent an unlimited combination of responses.
+
+
+Enabling Templating
+~~~~~~~~~~~~~~~~~~~
+
+By default templating is disabled. In order to enable it, set the flag to true in the response of a simulation.
+
+
+Available Data
+~~~~~~~~~~~~~~
+
+Currently, the following data is available through templating:
+
++------------------------------+----------------------------------------------+----------------------------------------------+--------+
+| Field                        | Example                                      | Request                                      | Result |
++==============================+==============================================+==============================================+========+
+| Request scheme               | {{ Request.Scheme }}                         | http://www.foo.com                           | http   |
++------------------------------+----------------------------------------------+----------------------------------------------+--------+
+| Query parameter value        | {{ Request.QueryParam.myParam }}             | http://www.foo.com?myParam=bar               | bar    |
++------------------------------+----------------------------------------------+----------------------------------------------+--------+
+| Query parameter value (list) | {{ Request.QueryParam.NameOfParameter.[1] }} | http://www.foo.com?myParam=bar1&myParam=bar2 | bar2   |
++------------------------------+----------------------------------------------+----------------------------------------------+--------+
+| Path parameter value         | {{ Request.PathParam.[1] }}                  | http://www.foo.com/zero/one/two              | one    |
```

Comment:
This should be `{{ Request.Path.[0] }}`. I think you can only get a path segment, not the path params.